lib/documents/schemas/product_safety_alerts.json

```diff
@@ -0,0 +1,144 @@
+{
+  "content_id": "a22b3f1f-2c91-49a1-a469-ff21d465c543",
+  "base_path": "/product-safety-alerts",
+  "format_name": "Product safety alerts, unsafe products and recalls",
+  "name": "Product safety alerts, unsafe products and recalls",
+  "description": "Find product safety alerts, unsafe products and recalls",
+  "filter": {
+    "format": "product_safety_alert"
+  },
+  "signup_content_id": "bb6047de-3854-4774-a5d5-fc66fed4f792",
+  "signup_copy": "You'll get an email each time an alert is updated or a new alert is published.",
+  "organisations": ["a0ee18e7-9e1e-4ba1-aed5-f3f287dce752"],
+  "topics": [],
+  "email_filter_by": "product_alert_type",
+  "email_filter_facets": [
```

Comment:
I've just looked at the associated card and it mentions that there should (?) be a subscription option for multiple facets, rather than just the 'Alert type', as per:

```
**Facets for the Finder:**
Users need to be able to filter and/or subscribe by :
- Alert Type
- Risk Level
- Product Category
- Measure Type
- Recall/alert date (before and after picker, like https://www.gov.uk/drug-device-alerts)
```

The comment isn't really clear on what subscription options are needed (and/or). Has there been an active choice here?

Reply:
Great spot - thanks @1pretz1! I've pushed up a commit that I think fixes that (lack of documentation, but I've explained my reasoning in the commit). Would be great to get a second pair of eyes on that.
 
I _think_ the new commit means we can close https://github.com/alphagov/email-alert-api/pull/1672 (based on https://docs.publishing.service.gov.uk/apps/specialist-publisher/creating-a-new-specialist-document-type.html#configure-the-email-sign-up-page) 🤔

Reply:
Ah, nice! The `all_selected_facets` option does seem less user friendly though as the user doesn't get a confirmation of what facets they want to subscribe to, for example https://www.gov.uk/residential-property-tribunal-decisions/email-signup (all facets option) vs https://www.gov.uk/cma-cases/email-signup. I think this happens due to the [logic in Finder Frontend](https://github.com/alphagov/finder-frontend/blob/main/app/presenters/subscriber_list_params_presenter.rb#L36-L38).

However, if want to enable email subscriptions for all the facets then this seems the only option. Hmm, should we ask BEIS to clarify if they actually want all facets, or just go with the `all_selected_facets`?

> I think the new commit means we can close alphagov/email-alert-api#1672

I believe you're correct. Could we also remove the `email_filter_facets` hash from this schema if we use `all_selected_facets`?

Reply:
Good shout. We're checking with BEIS. On hold until then.

Reply:
BEIS have decided to go with the standard `product_alert_type`.